Include/object.h

```diff
@@ -235,8 +235,12 @@ PyAPI_FUNC(void *) PyType_GetModuleState(struct _typeobject *);
 
 /* Generic type check */
 PyAPI_FUNC(int) PyType_IsSubtype(PyTypeObject *, PyTypeObject *);
-#define PyObject_TypeCheck(ob, tp) \
-    (Py_IS_TYPE(ob, tp) || PyType_IsSubtype(Py_TYPE(ob), (tp)))
+
+PyAPI_FUNC(int) PyObject_TypeCheck(PyObject *, PyTypeObject *);
+static inline int _PyObject_TypeCheck(PyObject *ob, PyTypeObject *tp) {
```

Comment:
Can you rename "tp" to "type"? Same name than in the doc.

(I'm fine with "ob" name.)